phoenix-core/src/main/java/org/apache/phoenix/mapreduce/index/IndexTool.java

```diff
@@ -235,15 +235,22 @@ public static IndexVerifyType fromValue(byte[] value) {
     private static final Option END_TIME_OPTION = new Option("et", "endtime",
```

Comment:
For consistency: These should both use "start-time" / "end-time" .. just like all other composite-word options.